contracts/orbital-core/src/account_types.rs

```diff
@@ -0,0 +1,56 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{ensure, DepsMut, StdError, StdResult, Uint64};
+
+use crate::state::OrbitalDomainConfig;
+
+#[cw_serde]
+pub enum AccountConfigType {
+    Polytone {
+        note: String,
+        timeout: Uint64,
+    },
+    ICA {
```

Comment:
you might want to add #[serde(rename = "ica")] here so that it doesn't get serialized/deserialized as i_c_a

Reply:
damn good catch, always forget this

Reply:
changed to `InterchainAccount`